### DISCUSSION TITLE
How can i disabled courier at kratos helm chart?

### DISCUSSION THREAD

**Discussion by iutx** [ory/kratos#2456] [Q&A] [answered]
Hi:
   We want to deploy kratos without courier component. But I can't find a configuration to disable it, thanks.

**Comment by vinckr** [comment, marked as answer]
Hello @iutx 
You are in luck, I believe this feature has just been contributed: https://github.com/ory/k8s/pull/550

**Reply by iutx** [reply]
I got it. Thanks!
